Remove canvas fallback

ovineq opened the pull request:
Remove non-WebGL fallback code in SchematicLayer since Chrome/Edge's Swift renderer (software CPU renderer) is fast enough and produces perfect results.

Issue: https://github.com/equinor/esv-intersection/issues/535

ooystein reviewed (changes requested):
Good job! The fallback implementation was not complete and delivered a sub optimal result compared to Chromium's Swift shader (CPU renderer). Now that we have improved render speed and removed the constant render loop we can fully rely on the CPU renderer for browsers/platforms without GPU.

Comment:
drawBigTexturedPolygon method can now be removed, it has no uses.

ovineq commented:
> Good job! The fallback implementation was not complete and delivered a sub optimal result compared to Chromium's Swift shader (CPU renderer). Now that we have improved render speed and removed the constant render loop we can fully rely on the CPU renderer for browsers/platforms without GPU.
> 
> Comment: drawBigTexturedPolygon method can now be removed, it has no uses.

Thanks and nice catch! I've removed the unused method.
